core/contracts/Voting.sol

```diff
@@ -223,29 +222,25 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
         uint currentRoundId = voteTiming.computeCurrentRoundId(blockTime);
 
         RequestStatus requestStatus = _getRequestStatus(priceRequest, currentRoundId);
-        if (requestStatus == RequestStatus.Active) {
-            return voteTiming.computeEstimatedRoundEndTime(currentRoundId);
-        } else if (requestStatus == RequestStatus.Resolved) {
-            return 0;
-        } else if (requestStatus == RequestStatus.Future) {
-            return voteTiming.computeEstimatedRoundEndTime(priceRequest.lastVotingRound);
-        }
 
-        // Price has never been requested.
-        // Price requests always go in the next round, so add 1 to the computed current round.
-        uint nextRoundId = currentRoundId.add(1);
-
-        priceRequests[priceRequestId] = PriceRequest({
-            identifier: identifier,
-            time: time,
-            lastVotingRound: nextRoundId,
-            index: pendingPriceRequests.length
-        });
-        pendingPriceRequests.push(priceRequestId);
-        emit PriceRequestAdded(nextRoundId, identifier, time);
-
-        // Estimate the end of next round and return the time.
-        return voteTiming.computeEstimatedRoundEndTime(nextRoundId);
+        if (
```

Comment:
What do you think about:
```
if (requestStatus == RequestStatus.NOT_REQUESTED) {
}
```

If you think that's better, we could also remove the `Price has never been requested` comment from below, since it's clear from the enum what case we're in.

Reply:
replaced with RequestStatus.NotRequested